report/renderer/report-utils.js

```diff
@@ -699,9 +377,12 @@ const UIStrings = {
   /** Label indicating that Lighthouse throttled the page using custom throttling settings. */
   runtimeCustom: 'Custom throttling',
 };
-Util.UIStrings = UIStrings;
+ReportUtils.UIStrings = UIStrings;
 
 export {
-  Util,
+  ReportUtils,
+  SharedUtils,
   UIStrings,
 };
+
+export * as MarkdownUtils from '../../shared/markdown.js';
```

Comment:
`git mv report/renderer/util.js shared/util.js` is a lot less churn/blame breakage and accomplishes essentially the same thing (and with less awkward naming/renaming). `util.js` is really already set up for this since it has no deps.

The markdown methods seem like the only part maybe crying out to be a separate file, though they've also been fine.

Reply:
IMO the churn is worth it because the weird report globals like `ReportUtils.reportJson` are now only visible to report code.

We can also add new dependencies to the report utils without worrying about the shared utils needing to remain standalone.

Reply:
> IMO the churn is worth it because the weird report globals like `ReportUtils.reportJson` are now only visible to report code.

that and `Util.i18n` are only there [due to failure of architecture and we're hiding our shame instead of putting them on `window`](https://github.com/GoogleChrome/lighthouse/blob/f4a1d10c3f8f6f6374b1e901cb3cfceda2020c59/report/renderer/report-renderer.js#L255-L261). They could go in or on any global variable.

> We can also add new dependencies to the report utils without worrying about the shared utils needing to remain standalone.

the shared utils still have to be standalone because they're still imported by the report.

Reply:
I'm unclear on how this could be just a simple `mv` of `report/renderer/util.js` -> `shared/util.js`, because of the existence of `Util.i18n` (soon to be `Util.strings` #13933). That results in a module in `shared/` being dependent on `report/renderer/i18n.js`. IIRC that's why I split the file.

I can rebase things and convince git this is a move, that would be good. I'll do a new PR, and defer the markdown changes, to make move simpler.

Reply:
annnnnd to reduce naming problems (so many utils), this seems like a good approach. just be honest about what we doing lol

<img width="522" alt="image" src="https://user-images.githubusercontent.com/4071474/213816266-1ff04392-0535-4545-a3fe-6c955026dc31.png">